.github/release.yml

```
@@ -0,0 +1,18 @@
+changelog:
+  exclude:
+    labels:
+      - ignore-for-release
+    authors:
+      - JohnnyQ5
```

This line was added in the release today to avoid github actions contributions in the changelog/release notes
```suggestion
      - JohnnyQ5
      - github-actions
```